cmd/compose/restart.go

```diff
@@ -40,6 +40,7 @@ func restartCommand(p *projectOptions, backend api.Service) *cobra.Command {
 		RunE: Adapt(func(ctx context.Context, args []string) error {
 			return runRestart(ctx, backend, opts, args)
 		}),
+		Args:              cobra.NoArgs,
```

Comment:
Ok I think we should update the documentation! Restart allow you to choose a specific service to restart so you can pass an argument like that:
```shell
docker compose restart my-service
```

Reply:
We shouldn't change this, right? It implies losing the ability to just restart a specific service, which people might be relying on.

Reply:
We'll keep the actual behavior for sure, that's why I propose to update the doc or remove the modification for the `restart` command

Reply:
Sorry, I had overlooked that `restart` could be used like that too.

So yes, let's update the docs.